pkg/util/ecs/types.go

```diff
@@ -43,6 +43,13 @@ type Network struct {
 	IPv4Addresses []string `json:"IPv4Addresses"` // one-element list
 }
 
+// Port represents the ports of a container
+type Port struct {
+	ContainerPort uint16
```

Comment:
Is there a particular reason not to specify JSON keys here? Otherwise I think it would be good for consistency to add them.

Reply:
yes JSON keys should be added, thanks!